model/src/main/kotlin/Project.kt

```diff
@@ -99,9 +99,18 @@ data class Project(
      * Contains dependency information as a [DependencyGraph]. This is an alternative format to store the dependencies
      * referenced by the various scopes. Use the [scopes] property to access dependency information independent on
      * the concrete representation.
+     * TODO: Remove this after all affected package managers have been converted to use a shared graph.
```

Comment:
Nit: Maybe leave an empty line to separate the TODO from the docs, to make it stand out more?